pkg/api/v1/routes/routes.go

```diff
@@ -149,14 +149,14 @@ func (r *Routes) Routes(g *gin.RouterGroup) {
 		servers.POST("/firmwareInstall", r.composeAuthHandler(createScopes("condition")),
 			wrapAPICall(r.firmwareInstall))
 
+		// Generalized API for any condition status (for cases where some server work
+		// has multiple conditions involved and the caller doesn't know what they might be)
+		servers.GET("/conditionStatus", r.composeAuthHandler(readScopes("condition")),
+			wrapAPICall(r.conditionStatus))
```

Comment:
Not sure how much we care about this, but I think "/conditions" would be a more RESTful way to express an endpoint that returns a collection of conditions.

Reply:
That's a good point, and I do like being super RESTful; my only misgiving with that is that this isn't a general purpose condition history endpoint. It's intended as more of "what's going on with this server now (or in the immediate past)."

That said, if we get a consensus for change, I'm not opposed to it either.

Reply:
> It's intended as more of "what's going on with this server now (or in the immediate past)."

nit pick: I think this could be just `/status`, so we drop the `condition` prefix.